main/src/ca/uwaterloo/flix/language/ast/Symbol.scala

```diff
@@ -413,7 +414,14 @@ object Symbol {
   /**
     * Restrictable Enum Symbol.
     */
-  final class RestrictableEnumSym(val namespace: List[String], val name: String, val loc: SourceLocation) extends Symbol {
+  final class RestrictableEnumSym(val namespace: List[String], val name: String, cases: List[Name.Ident], val loc: SourceLocation) extends Symbol {
```

Comment:
I think we want to override equality... No?

Reply:
Ah, we already have the right equality, correct?

Reply:
Correct